MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+include *.txt
+include README.rst
```

Comment:
Would it make sense to include `CHANGES.txt` and `LICENSE.txt`, too?  I notice that the traits `MANIFEST.in` also includes `MANIFEST.in`, but I'm not sure whether that's necessary.